doc/reference/building-and-signing-tx.md

```diff
@@ -10,84 +10,82 @@ Transactions vary in complexity, depending on their intended outcomes, but all t
 
-To create a registration certificate, use the following code:
+    cardano-cli shelley query protocol-parameters \
 
-`cardano-cli shelley stake-address registration-certificate \
-     --staking-verification-key-file stake.vkey \
-     --out-file stake.cert`
+    --shelley-mode           For talking to a node running in Shelley-only mode
+                             (default).
+    --byron-mode             For talking to a node running in Byron-only mode.
+    --epoch-slots NATURAL    The number of slots per epoch (default is 21600).
+    --security-param NATURAL The security parameter (default is 2160).
+    --cardano-mode           For talking to a node running in full Cardano mode.
+    --epoch-slots NATURAL    The number of slots per epoch (default is 21600).
+    --security-param NATURAL The security parameter (default is 2160).
+    --mainnet                Use the mainnet magic id.
+    --testnet-magic NATURAL  Specify a testnet magic id.
+    --out-file FILE          Optional output file. Default is to write to stdout.
 
-A deposit amount is specified in the protocol.json file, under keyDeposit. This amount has to be included in the transaction containing the certificate, but will be refunded when the key is deregistered.
+**Signing a transaction**
 
-`"keyDeposit": 400000,`
+A transaction must prove that it has the right to spend its inputs. In the most common case, this means that a transaction must be signed by the signing keys belonging to the payment addresses of the inputs.
+If a transaction contains certificates, it must additionally be signed by somebody with the right to issue those certificates. For example, a stake address registration certificate must be signed by the signing key of the corresponding stake key pair.
+
+    cardano-cli shelley transaction sign \
+    --tx-body-file FILE      Input filepath of the TxBody.
+    --signing-key-file FILE  Input filepath of the signing key (one or more).
+    --mainnet                Use the mainnet magic id.
+    --testnet-magic NATURAL  Specify a testnet magic id.
+    --out-file FILE          Output filepath of the Tx.
+
+**Submit a transaction**
+
+Submitting a transaction means sending the signed transaction through the local node whose Unix domain socket is obtained
+from the CARDANO_NODE_SOCKET_PATH enviromnent variable.
+
+    cardano-cli shelley transaction submit
+
+    --shelley-mode           For talking to a node running in Shelley-only mode
+                             (default).
+    --byron-mode             For talking to a node running in Byron-only mode.
+    --epoch-slots NATURAL    The number of slots per epoch (default is 21600).
+    --security-param NATURAL The security parameter (default is 2160).
+    --cardano-mode           For talking to a node running in full Cardano mode.
+    --epoch-slots NATURAL    The number of slots per epoch (default is 21600).
```

Comment:
if issue #1451  is resolved, this will need updating

Reply:
noted